setup.py

```diff
@@ -18,7 +18,7 @@
 
 install_requires = [
     "requests>=2.30.0,<3.0",
-    "urllib3>=2.0.0,<3.0",
+    "urllib3>=1.25.10",
```

Comment:
```suggestion
    "urllib3>=1.25.10,<3.0",
```

We should keep the upper bound. Future versions of urllib3 may require changes in responses.

Reply:
Thanks. Done.